### PR TITLE
hello world nick

### LINKED ISSUE #20: Hello World Nick
- create a new entity in the commons package which is called `<your-name>Pet` (exp: `BobsPet`)
- create a backend service which serves this entity (save the entity in memory, simple variable)
  - [Docs](https://docs.feathersjs.com/api/services.html)
- create a vue component which
  - triggers a call to your new backend service (for example by pressing a button) and creates an entity
  - shows a list of your entities (initial find method call) and adds newly created entities [`created` event](https://docs.feathersjs.com/api/events.html) to that list

### PR DESCRIPTION
**Pull request by anbraten**
closes #20

### CODE REVIEW

File: packages/backend/src/services/index.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { Application } from '../declarations';
 import users from './users/users.service';
-// Don't remove this comment. It's needed to format import lines nicely.
+import nicksPet from './nicksPet/nicksPet.service';
```
Comment (JanEberlein) [inline packages/backend/src/services/index.ts:3]: Is there a reason why you removed the comment that specifically asks you not to remove it?

File: packages/commons/package.json [hunk 2]
```diff
@@ -12,5 +12,7 @@
     "reset": "yarn clean"
   },
   "dependencies": {
+    "@types/uuid": "^8.3.0",
+    "uuid": "^8.3.1"
```
Comment (JanEberlein) [inline packages/commons/package.json:16]: Do you still need these?

**Review by JanEberlein — COMMENTED** [review]
Only a few questions

File: packages/app/src/components/HelloWorld.vue [hunk 3]
```diff
@@ -1,45 +1,17 @@
...
-
-    onMounted(async () => {
-      // users.value = await feathers.service('users').find();
-    });
-
-    return {
-      count,
-      users,
-    };
+  components: {
```
Comment (julianhahn) [inline packages/app/src/components/HelloWorld.vue:13]: You use your HelloWorld component only to create your pet component.
Maybe create your pet component directly in the App.Vue.

File: packages/app/src/components/NicksPetList.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="box-content bg-blue-500 m-2 p-6">
```
Comment (julianhahn) [inline packages/app/src/components/NicksPetList.vue:2]: Just a personal preference. With the max-w the blue container does not span the whole Screen. Looks weird on an ultrawide.

```suggestion
  <div class="box-content mx-auto max-w-screen-xl bg-blue-500 m-2 p-6">
```

File: packages/commons/package.json [hunk 5]
```diff
@@ -12,5 +12,7 @@
     "reset": "yarn clean"
   },
   "dependencies": {
+    "@types/uuid": "^8.3.0",
```
Comment (julianhahn) [inline packages/commons/package.json:15]: I would suggest to remove those two uuid because an unique id is auto-generated by feathers.

**Review by julianhahn — APPROVED** [review]
super crisp UI with remove function, nice!

File: packages/backend/src/services/index.ts (hunk 1 above)
Comment (JanEberlein) [reply packages/backend/src/services/index.ts:4]: that comment isn't needed

File: packages/commons/package.json (hunk 5 above)
Comment (anbraten) [reply packages/commons/package.json]: resolved by 0bcdd865854bae3b21873da2fb3d60762f3e19af

File: packages/commons/package.json (hunk 2 above)
Comment (anbraten) [reply packages/commons/package.json]: resolved by 0bcdd865854bae3b21873da2fb3d60762f3e19af

File: packages/backend/src/services/index.ts (hunk 1 above)
Comment (anbraten) [reply packages/backend/src/services/index.ts:4]: It was auto-generated by feathers-cli. No problem to remove it.

**Review by JanEberlein — APPROVED** [review]

**Pull request merged**